website/docs/cloud-docs/integrations/service-now/service-graph/resource-coverage.mdx

```diff
@@ -0,0 +1,611 @@
+---
```

Comment:
Would we want to break this up and make separate pages for each cloud provider? It might future proof the pages a little too as more resources come along

Reply:
I like this idea. Will do!